### ISSUE TITLE
Firewall rules excessive changes on terraform plan

### ISSUE THREAD

**Issue by markgov** [ministryofjustice/modernisation-platform#5431]
### User Story

As a Modernisation platform engineer
I want to explore a better way of merging all the firewall json file together
So that when a plan is run it only shows the new rule that is added and dose not change the position of existing firewall rules

### Value / Purpose

This was noticed by one of our account holders when he put a new PR in for new firewall rules when he looked at the plan he noticed a lot of changes happening and could not tell if his changes where going though, it was only after investigation that we found out that this was expected behaviour due to the random nature of how merge works in terraform. That it recreates the main firewall file and current firewall positions are not guaranteed. 

### Useful Contacts

Mark Roberts

### Definition of Done

- [x] Alternative options explored
- [x] Potentially implement new option
- [x] Another team member has reviewed
- [x] Firewall rules can be applied without significant churn

**Comment by dms1981** [comment]
I don't think there's a better way on this; because the rules.json files are merged and passed in as an object - `aws_networkfirewall_rule_group` - there will be changes to that resource, and those will be output into the Terraform plan.

**Comment by markgov** [comment]
potenial change
https://github.com/ministryofjustice/modernisation-platform/pull/8774/files
adds a sort before the merge

**Comment by Khatraf** [comment]
DoD has been ticked off, and the solution works as expected so moving it to done.
